shell/components/ResourceDetail/Masthead.vue

```diff
@@ -609,10 +609,10 @@ export default {
   }
 
   HEADER {
-    margin: 0 0 0 -5px;
+    margin: 0;
 
     .title {
-      overflow-x: hidden;
+      overflow-x: visible;
```

Comment:
This fixes the focus issue :

![image](https://github.com/user-attachments/assets/c42701dd-6d30-496f-8c28-ff69429519ed)

It's not clear why we wanted to hide the overflow here. Everything I tried still seems to work. @aalves08 Is there an edge case you're aware of?

Reply:
not really, no... I am afraid of that `overflow`, that's why I didn't mess with it 😅 😅 😅

Reply:
Ah, you just tweaked it to be x only https://github.com/rancher/dashboard/commit/c6db1b70dfd5a4a7b252585e7904ff45b59c3e0f#diff-dcf43af1960ee931344da2d37e6237c086acbdacc5d16489ec02e67319c27338R593

So was the focus getting cut off on the top too?

Reply:
I don't think so, but you can double-check that. If it doesn't on your PR, then we 🚢 it 😛

Reply:
I found the original reason we added overflow https://github.com/rancher/dashboard/pull/10295/files#diff-dcf43af1960ee931344da2d37e6237c086acbdacc5d16489ec02e67319c27338R550.

I'll make sure that's working as well.